packages/terra-dropdown-button/src/_Button.jsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import styles from './_Button.module.scss';
+
+const cx = classNames.bind(styles);
+
+const propTypes = {
+  /**
+   * The text that is displayed on the option.
+   */
+  label: PropTypes.string.isRequired,
+  /**
+   * The function that is called when the option is clicked or enter/space is pressed with it focused.
+   */
+  onClick: PropTypes.func.isRequired,
+  /**
+   * @private
+   * Whether or not the option should show active styles
+   */
+  isActive: PropTypes.bool,
+  /**
+   * @private
+   * Whether or not the option should show keyboard focus styles
+   */
+  isFocused: PropTypes.bool,
+  /**
+   * @private
+   * Callback to tell the parent it should close the dropdown
+   */
+  requestClose: PropTypes.func,
+};
+
+const Option = ({
+  label, onClick, isFocused, isActive, requestClose, ...customProps
+}) => (
+  <li>
+    <div
+      {...customProps}
```

Comment:
Custom props probably shouldn't cascade this far since it's an internal component

Reply:
Custom props don't cascade down to here, this would only be what the user of the component passes directly to the `Button`. Thinking about it, this really isn't an internal component and probably shouldn't have an underscore prefixed name.